app/controllers/amends/AmendHumanMedicinePlasticPackagingController.scala

```diff
@@ -59,12 +56,8 @@ class AmendHumanMedicinePlasticPackagingController @Inject() (
       implicit request =>
         val pptId: String = request.pptReference
 
-        val obligation = request.userAnswers.get[TaxReturnObligation](ObligationCacheable).getOrElse(
```

Comment:
if(request.userAnswers.get[TaxReturnObligation](ObligationCacheable).isEmpty()) {
          throw new IllegalStateException("Must have a tax return against which to amend")
}